File: test_regex.py
import re
text = "MUYUNGAKLKTIMOTHY<<K<KLKLKLKLLKLKLLKLKLKL"
new_text = re.sub(r'([A-Z]{3,})(LKK|KKL|LKL|KKK|LLL|KLK|LKC|KLC)([A-Z]{3,})', r'\1<<\3', text)
print("Replaced:", new_text)

File: test_regex.js
const text = "MUYUNGAKLKTIMOTHY<<K<KLKLKLKLLKLKLLKLKLKL";
const newText = text.replace(/([A-Z]{3,})(LKK|KKL|LKL|KKK|LLL|KLK|LKC|KLC)([A-Z]{3,})/g, "$1<<$3");
console.log("Replaced:", newText);
